File: TOKI-Learning/pascal/if_then_case.pas
var
	a : longint;
begin
	readln(a);
	case a of 
	1..9 : writeln('satuan');
	10..99 : writeln('puluhan');
	100..999 : writeln('ratusan');
	1000..9999 : writeln('ribuan');
	10000..99999 : writeln('puluhribuan');
	end;
end.

File: TOKI-Learning/c++/if_then_case.cpp
#include <cstdio>

int main()
{
    int n;
    scanf("%d", &n);
    if (n < 10)
        printf("satuan");
    else if (n < 100)
        printf("puluhan");
    else if (n < 1000)
        printf("ratusan");
    else if (n < 10000)
        printf("ribuan");
    else
        printf("puluhribuan");
    printf("\n");
    return 0;
}
